ledgers/xero/lib/ledger_sync/ledgers/xero/client.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require_relative 'oauth_client'
+
+module LedgerSync
```

Comment:
How similar is this to the QBO code?  Could we DRY this up by moving some of this to the core or to another class that handles authentication?